src/application.rs

```diff
@@ -94,6 +100,22 @@ impl SwayOSDApplication {
 			"Shows capslock osd. Note: Doesn't toggle CapsLock, just display the status",
 			None,
 		);
+		app.add_main_option(
+			"num-lock",
+			glib::Char::from(0),
+			OptionFlags::NONE,
+			OptionArg::None,
+			"Shows numlock osd. Note: Doesn't toggle NumLock, just display the status",
```

Comment:
Oops, we should change all of the "just display" to "just displays" for all of the options. Mb